doc/sphinx-guides/source/admin/ror.rst

```diff
@@ -0,0 +1,58 @@
+ROR
+===
+
+.. contents:: Contents:
+    :local:
+
+About ROR
+---------
+
+ROR (Research Organization Registry) is a registry of identifiers for research
+organizations. ROR identifiers helps with matching affiliation data with institutions.
+
+For more informations visit `official ROR homepage <https://ror.org/>`_
+
+
+ROR in Dataverse
+----------------
+
+Dataverse uses ROR identifiers to enrich DataCite metadata format. In default dataset metadata fields
+configuration there is also turned on suggesting of ROR identifiers based on name of the organization.
+
+
+Uploading ROR data
+------------------
+
+To fully profit from ROR integration in Dataverse you will need to feed your Dataverse installation with ROR data.
+You can either use full ROR data dump with all available ROR identifiers or a subset of it.
+
+Full dump
+~~~~~~~~~
+
+First you will need to obtain ROR dump by following instructions presented in `ROR data dump documentation <https://ror.readme.io/docs/data-dump>`_.
+
+The next step is to upload obtained ROR dump into your Dataverse installation. It can be done using
+the following REST api endpoint::
+
+        curl -H "X-Dataverse-key:$API_TOKEN" -F file=@2021-03-25-ror-data.zip "http://localhost:8080/api/ror/upload"
+
+Note that you will need superuser permissions to upload ROR data.
+
+Partial dump
+~~~~~~~~~~~~
+
+It is possible to use the same REST api enpoint as in full dump case to upload only selected ROR identifiers.
+In such cases you will need to prepare the dump data yourself. We will show an example how it can be achieved.
+
+Dump can be prepared using ROR api:
+
+.. literalinclude:: ../_static/util/make_ror_dump.sh
+
+In example above we include ROR identifiers for institutions from `Great Britain` with type `Education`.
```

Comment:
Maybe add a sentence like: `Modify make_ror_dump.sh to obtain the data you want`.

Reply:
I will change it to: `You can modify ``ROR_FILTER`` variable in the script to obtain the data you want.` (I don't include filename because the script is embedeed on the page - user don't have a file it can download; he must do a copy+paste to use it)